src/modules/addRemove.js

```diff
@@ -1,4 +1,5 @@
 import Task from './task.js';
+import taskCompleted from './removeCompleted.js';
 
 class displayTask {
   static displayTasks() {
```

Comment:
- [ ] Kindly ensure that a task remains checked when the browser is reloaded. This means that you need to update the localStorage when a task is checked.

Reply:
@oloomoses The required change has been implemented